src/components/Banner/BannerMobile.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Dialog } from '@santiment-network/ui'
+import styles from './BannerMobile.module.scss'
+
+const classes = { title: styles.title }
```

Comment:
`styles` is already an object which contains `title` property

Reply:
I didn't understand( 
You mean write: 
` <Dialog trigger={<></>} position='bottom' defaultOpen classes={styles.title}>`
 without creating object for that?

that doesn't work for me without object

Reply:
`classes={styles}`

Reply:
oh, yes, my mistake